<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DepositCalcView</class>
 <widget class="QWidget" name="DepositCalcView">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1065</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1065</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1065</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Deposit Calculator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: #191A19;
	color: white;
	font-family: Sans-serif;
	font-size: 14px;
}

QLineEdit {
	border: 1px solid #606060;
	border-radius: 25px;
	padding-left: 10px;
}

QTextBrowser {
	padding-left: 10px;
}

QPushButton {
	border: 1px solid #606060;
	border-radius: 25px;
}

QPushButton::hover {
	background-color: #444444;
}

QPushButton::pressed {
	background-color: #282828;
}

QSpinBox {
	border: 1px solid #606060;
	border-radius: 25px;
	padding-left: 10px;
}

QDoubleSpinBox {
	border: 1px solid #606060;
	border-radius: 25px;
	padding-left: 10px;
}

QComboBox {
	border: 1px solid #606060;
	border-radius: 25px;
	padding-left: 10px;
}
QComboBox::item::selected {
	background-color: #444444;
}

QDateEdit {
	border: 1px solid #606060;
	border-radius: 20px;
	padding-left: 10px;
}</string>
  </property>
  <widget class="QPushButton" name="homeBtn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>400</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: #374045;
}

QPushButton::hover {
	background-color: #3A4750;
}

QPushButton::pressed {
	background-color: #303841;
}</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="amountInputWidget">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>40</y>
     <width>301</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans-serif</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="showGroupSeparator" stdset="0">
    <bool>true</bool>
   </property>
   <property name="decimals">
    <number>2</number>
   </property>
   <property name="minimum">
    <double>0.010000000000000</double>
   </property>
   <property name="maximum">
    <double>10000000000.000000000000000</double>
   </property>
   <property name="value">
    <double>100000.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="amountLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans-serif</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Deposit amount</string>
   </property>
  </widget>
  <widget class="QComboBox" name="periodTypeWidget">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>40</y>
     <width>111</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans-serif</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:0;</string>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentText">
    <string>years</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="maxVisibleItems">
    <number>2</number>
   </property>
   <property name="insertPolicy">
    <enum>QComboBox::InsertAtBottom</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
   </property>
   <property name="minimumContentsLength">
    <number>0</number>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
   <property name="duplicatesEnabled">
    <bool>false</bool>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="modelColumn">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>years</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>months</string>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="periodInputWidget">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>40</y>
     <width>191</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans-serif</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-top-right-radius: 0;
border-bottom-right-radius: 0;
</string>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>50</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="periodLabel">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>50</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans-serif</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Period</string>
   </property>
  </widget>
  <widget class="QLabel" name="rateLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans-serif</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Interest rate</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="rateInputWidget">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>130</y>
     <width>301</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans-serif</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="showGroupSeparator" stdset="0">
    <bool>true</bool>
   </property>
   <property name="decimals">
    <number>2</number>
   </property>
   <property name="minimum">
    <double>0.010000000000000</double>
   </property>
   <property name="maximum">
    <double>1000.000000000000000</double>
   </property>
   <property name="value">
    <double>5.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="frequencyLabel">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>150</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans-serif</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Frequency</string>
   </property>
  </widget>
  <widget class="QComboBox" name="frequencyTypeWidget">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>140</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans-serif</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:0;</string>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentText">
    <string>Everyday</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="maxVisibleItems">
    <number>7</number>
   </property>
   <property name="insertPolicy">
    <enum>QComboBox::InsertAtBottom</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
   </property>
   <property name="minimumContentsLength">
    <number>0</number>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
   <property name="duplicatesEnabled">
    <bool>false</bool>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="modelColumn">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>Everyday</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Monthly</string>
    </property>
   </item>
  </widget>
  <widget class="QCheckBox" name="capitalCheck">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>210</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox::indicator {
	border: 1px solid white;
}

QCheckBox::indicator:checked {
	border: none;
	background-color: #E16428;
}

QCheckBox::indicator:unchecked {
	border: 1px solid white;
	background-color: rgb(61, 56, 70);
}

</string>
   </property>
   <property name="text">
    <string>Interest capitalization</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
   <property name="tristate">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="equalBtn">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>400</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: #E16428;
}

QPushButton::hover {
	background-color: #ff8000;
}

QPushButton::pressed {
	background-color: #b35900;
}</string>
   </property>
   <property name="text">
    <string>Calculate</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="interestOutputWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>510</y>
     <width>181</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans-serif</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="showGroupSeparator" stdset="0">
    <bool>true</bool>
   </property>
   <property name="decimals">
    <number>2</number>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>10000000000.000000000000000</double>
   </property>
   <property name="value">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="interestOutputLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>520</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans-serif</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Interest charges</string>
   </property>
  </widget>
  <widget class="QLabel" name="taxInfoLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>461</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans-serif</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	border: 1px solid white;
}</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The key rate of the Central Bank for 2022 is 11%. Therefore, the tax-free amount is 110 000.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="taxOutputWidget">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>510</y>
     <width>211</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans-serif</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="showGroupSeparator" stdset="0">
    <bool>true</bool>
   </property>
   <property name="decimals">
    <number>2</number>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>10000000000.000000000000000</double>
   </property>
   <property name="value">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="taxLabel">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>520</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans-serif</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Tax</string>
   </property>
  </widget>
  <widget class="QLabel" name="depositAmountLabel">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>500</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans-serif</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Deposit amount by the end of the term&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="depAmountOutputWidget">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>510</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans-serif</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="locale">
    <locale language="English" country="Kenya"/>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="showGroupSeparator" stdset="0">
    <bool>true</bool>
   </property>
   <property name="decimals">
    <number>2</number>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>10000000000.000000000000000</double>
   </property>
   <property name="value">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="tableBtn">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>610</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #414141;</string>
   </property>
   <property name="text">
    <string>Payment schedule</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearBtn">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>400</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: #1A1A2E;
}

QPushButton::hover {
	background-color: #0F3460;
}

QPushButton::pressed {
	background-color: #0A043C;
}</string>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="replBtn">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>300</y>
     <width>251</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">text-align: center;</string>
   </property>
   <property name="text">
    <string>Add replenishment</string>
   </property>
  </widget>
  <widget class="QPushButton" name="withdrawBtn">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>300</y>
     <width>251</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">text-align: center;</string>
   </property>
   <property name="text">
    <string>Add withdrawal</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
